src/map_tiles/apps.py

```diff
@@ -0,0 +1,41 @@
+from django.apps import AppConfig
+import os
+import zipfile
+
+
+class MapTilesConfig(AppConfig):
+    name = "map_tiles"
+
+    def ready(self):
+        """
+        This method is called when the Django app registry is fully populated.
+        It's a suitable place to perform initialization tasks.
+        """
+        # Prevent running initialization multiple times during development
+        if os.environ.get("RUN_MAIN") != "true":
+            return
```

Comment:
Probably can get rid of this, my current setup always returns this as None anyways.